docs/book/src/topics/azure-cni-v1.md

```diff
@@ -0,0 +1,138 @@
+# Using Azure CNI V1
+
+This document provides step to use Azure CNI as the CNI solution in your workload cluster.
+
+## Azure Container Networking Interface v1
+
+While following the [quick start steps in Cluster API book](https://cluster-api.sigs.k8s.io/user/quick-start.html#quick-start), Azure CNI can be used in place of Calico as a container networking interface solution for your workload cluster.
+
+Artifacts required for Azure CNI:
+
+- [azure-cni.yaml](https://raw.githubusercontent.com/Azure/azure-container-networking/v1.5.3/hack/manifests/cni-installer-v1.yaml)
+
+<!-- TODO: will we be exposing azure cni v1 customized cluster manifest for the users to use? If no, I will put in a example manifest here for the users. -->
+
+## Update Cluster Configuration
+
+The following resources need to be updated if working off of `capi-quickstart.yaml` (the default flavored cluster manifest generated while following the Cluster API quick start).
+
+- `kind: AzureCluster`
+  - update `spec.networkSpecs.subnets` with the name and role of the subnets you want to use in your workload cluster.
+
+  - ```yaml
+    apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+    kind: AzureCluster
+    metadata:
+      name: ${CLUSTER_NAME}
+      namespace: default
+    spec:
+      .
+      .
+      networkSpec:
+        subnets:
+        - name: control-plane-subnet # update this as per your nomenclature
+          role: control-plane
+        - name: node-subnet # update this as per your nomenclature
+          role: node
+      .
+      .
+    ```
+
+- `kind: KubeadmControlPlane` of control plane nodes
+  - add `max-pods: "110"` to `spec.kubeadmConfigSpec.initConfiguration.nodeRegistration.kubeletExtraArgs`.
```

Comment:
I wonder if we should make the example use `30` max pods instead of 110 to follow the AKS default https://learn.microsoft.com/en-us/azure/aks/configure-azure-cni#maximum-pods-per-node

Reply:
(and if so we should also update the privateIPConfigs to 30)

Reply:
I agree to both, updating the private IP limit to default of `30`